calculator.js

```diff
@@ -1,5 +1,40 @@
+//Object
 const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
+  num1: 4,
+  num2: 4,
+  operation: 'Add',
 }
+
+// Calculator.
+const calculator = function(input){
+
+  if(input.operation === "Add" || input.operation === "+"){
+    return input.num1 + input.num2;
+  }else if (input.operation === "Subtract" || input.operation === "-"){
+    return input.num1 - input.num2;
+  }else if (input.operation === "Multiply" || input.operation === "*"){
+    return input.num1 * input.num2;
+  }else if (input.operation === "Divide" || input.operation === "/"){
+    let result = input.num1 / input.num2;
```

Comment:
Ooops

```suggestion
    return input.num1 / input.num2;
```

Reply:
I return the result of the Divide. Thanks a lot.